File: Linguagem C/exercicios de aula/menor_de_tres.c
#include <stdio.h>

int main()
{
    int n1,n2,n3;

    printf("Primeiro valor: ");
    scanf("%d", &n1);

    printf("Segundo valor:o: ");
    scanf("%d", &n2);

    printf("Terceiro valoro: ");
    scanf("%d", &n3);

    if (n1 < n2 && n1 < n3)
    {
        printf("MENOR = %d\n",n1);
    }
    else    if (n2 < n1 && n2 < n3)
    {
        printf("MENOR = %d\n",n2);
    }
    else    if (n3 < n1 && n3 < n2)
    {
        printf("MENOR = %d\n",n2);
    }
    else
    {
        printf("MENOR = %d\n",n3);
    }

    return 0;

}

File: Python/exercicios propostos/menor_de_tres.py
num1 = int(input("Primeiro valor: "))
num2 = int(input("Segundo valor: "))
num3 = int(input("Terceiro valor: "))

if num1 < num2 and num1 < num3:
  print(f"MENOR = {num1}")
elif num2 < num1 and num2 < num3:
  print(f"MENOR = {num2}")
else:
  print(f"MENOR = {num3}")